src/test/java/com/brandwatch/internship/fetchandstoretweets/services/QueryServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@TestPropertySource(locations="classpath:test.properties")
```

Comment:
For this test we don't need any of these annotations, we don't use Spring dependency injection here, nor should we do calls to the database or http server.

Reply:
What you may need is to generate and inject mocks 
```
@RunWith(MockitoJUnitRunner.class)
```

Reply:
This one causes a problem that I expected to happen. It is related to the next comment you made about anyLong() so I will explain it there.